build.gradle

```diff
@@ -4,9 +4,10 @@ buildscript {
     }
 
     dependencies {
-        classpath 'com.android.tools.build:gradle:2.0.0-beta5'
+        classpath 'com.android.tools.build:gradle:2.1.0-alpha4'
```

Comment:
Yes, please do. If you're using an unstable version of Android Studio, it will suggest an unstable version of gradle. I'd prefer to keep this as stable as possible.